types/src/shared/utils/structured_data.ts

```diff
@@ -14,7 +14,7 @@ export function getSanitizedHeaders(rawHeaders: string[]) {
       acc.push(slugifiedName);
     } else {
       let conflictResolved = false;
-      for (let i = 2; i < 64; i++) {
+      for (let i = 2; i < 1024; i++) {
```

Comment:
This seems like high impact?

Reply:
This should not be needed IMO. If we have 64 columns with the same header, there's a problem (and we likely cannot do much with the CSV in the first place)

Reply:
I had the issue with a table with about ~80 columns, all of them without data on the first row (this one: https://docs.google.com/spreadsheets/d/123XObNDcnbMcrwSs2YShLZlL_fjV6fnHt11TU4MvGSE/edit?gid=7921452#gid=7921452 ) . If we're using the app it's working ok, as all column names are filled (with a few repetitions), but if we rely on static non-app headers detection, we have a lot of "unknown" columns and 64 is not enough, and this makes the activity fails. 
Btw even if we have the app enabled, i'm still calling rowsFromCsv with app disabled for validation before upserting the activity.
Otherwise, I agree that if we have more than 64 columns with the same name we won't be able to do much of the table.